source/extensions/filters/network/thrift_proxy/conn_manager.cc

```diff
@@ -176,6 +176,15 @@ DecoderEventHandler& ConnectionManager::newDecoderEventHandler() {
   return **rpcs_.begin();
 }
 
+bool ConnectionManager::passthroughEnabled() const {
+  if (!config_.payloadPassthrough()) {
+    return false;
+  }
+
+  ASSERT(!rpcs_.empty());
```

Comment:
Nit: can you add some comment to explain why this assertion holds true?

Reply:
Done.